Short-Answer/Algorithms_Answers.md

```diff
@@ -1 +1,46 @@
-Add your answers to the Algorithms exercises here.
\ No newline at end of file
+Add your answers to the Algorithms exercises here.
+
+```
+a)  a = 0
+    while (a < n * n * n):==> O(n)
+      a = a + n * n ========> O(1)
+```
+
+`This is O(n) because every time n is incresed the run time grows.`

```

Comment:
Wait, I mean, correct answer, but this is a little generic lol. Perhaps you should mention how some of those `n`s are nullfied by the `a = a + n * n`? hahaha but great job either way!